modules/files/README.md

```diff
@@ -1,27 +1,25 @@
 # `files`
 
 The `files` module can be used to copy directories from `files/` to
-any location in your image at build time, as long as the location exists at
-build time (e.g. you can't put files in `/home/<username>/`, because users
+any location in your image at build-time, as long as the location exists at
+build-time (e.g. you can't put files in `/home/<username>/`, because users
 haven't been created yet prior to first boot).
 
 :::note
-If you want to place files into `/etc/`, there are two ways to do it:
+Don't copy files directly to `/usr/etc/` in build-time, but copy those to `/etc/` instead,
+due to the nature of how `ostree` handles `/usr/etc/` & `/etc/` relationship.
```

Comment:
I don't know if explaining this is required. Writing files to /etc/ is pretty ordinary. Might want to add a note that "previously this page recommended writing files to /usr/etc/, but now...".

Reply:
I think it's needed, since there would be people who would ask about why we copy to `/etc/` instead of `/usr/etc/`

Reply:
That would be in the announcement for people who are currently using /usr/etc/, but if someone has never heard of that directory, why confuse them with it? Further explanation _could_ be added somewhere, like the "Building on Universal Blue" page I guess..

Reply:
I only left this now:

```
In run-time, `/usr/etc/` is the directory for "system"
configuration templates on atomic Fedora distros, whereas `/etc/` is meant for
manual overrides and editing by the machine's admin *after* installation.
```

I find this information useful for the users.

Is that alright for you?

Reply:
This doesn't make it clear if users should put their `/etc/` edits in `/etc/` or `/usr/etc/`. It's not clear to me personally either right now, what about you?

If `/etc/` is the correct place:
> Might want to add a note that "previously this page recommended writing files to /usr/etc/, but now...".

Reply:
I explicitly mention `In run-time`, but I guess that things can be explained better.

`/etc/` can't be edited in build-time, as editing `/etc/` is technically editing `/usr/etc/` in proper way according to `ostree` documentation.

Reply:
I clarified it better now.